test/data/adult-test.txt

```diff
@@ -0,0 +1,119 @@
+0	Private	Bachelors	Never-married	Sales	Own-child	White	Female	United-States	25	196947	13	0	0	40
+0	Self-emp-not-inc	HS-grad	Married-civ-spouse	Exec-managerial	Husband	White	Male	United-States	41	233150	9	0	0	60
+0	Private	Some-college	Never-married	Sales	Own-child	White	Female	United-States	18	42857	10	0	0	20
+1	Private	Doctorate	Married-civ-spouse	Prof-specialty	Husband	White	Male	United-States	44	118059	16	0	0	55
+0	Private	Some-college	Married-civ-spouse	Exec-managerial	Husband	White	Male	United-States	40	169262	10	3411	0	50
+0	Private	Bachelors	Never-married	Prof-specialty	Not-in-family	White	Male	United-States	27	95108	13	0	0	40
+0	Private	Bachelors	Never-married	Adm-clerical	Own-child	White	Male	United-States	23	161092	13	0	0	40
+0	Private	Some-college	Never-married	Adm-clerical	Not-in-family	Other	Male	United-States	21	345253	10	2174	0	40
+0	State-gov	HS-grad	Married-civ-spouse	Transport-moving	Husband	White	Male	United-States	37	111275	9	0	0	40
+0	Self-emp-not-inc	HS-grad	Married-civ-spouse	Farming-fishing	Husband	White	Male	United-States	65	178878	9	0	0	48
+1	Federal-gov	Masters	Married-civ-spouse	Prof-specialty	Husband	White	Male	United-States	43	157237	14	0	0	40
+0	Private	HS-grad	Never-married	Adm-clerical	Not-in-family	White	Male	United-States	45	155664	9	0	0	44
+0	State-gov	Bachelors	Never-married	Exec-managerial	Not-in-family	White	Female	United-States	24	322658	13	0	0	40
+0	Private	Some-college	Never-married	Machine-op-inspct	Not-in-family	White	Male	United-States	24	208503	10	0	0	42
+0	Local-gov	Some-college	Never-married	Protective-serv	Own-child	White	Male	United-States	19	223326	10	0	1721	35
+1	Private	Bachelors	Married-civ-spouse	Prof-specialty	Husband	White	Male	United-States	37	20308	13	0	0	65
+0	Private	HS-grad	Never-married	Adm-clerical	Own-child	White	Female	United-States	24	124751	9	0	0	35
+0	Private	Some-college	Married-civ-spouse	Machine-op-inspct	Husband	White	Male	Germany	34	113364	10	0	0	40
+0	State-gov	HS-grad	Married-civ-spouse	Other-service	Wife	White	Female	United-States	50	196900	9	0	0	39
+0	Private	HS-grad	Married-civ-spouse	Craft-repair	Husband	White	Male	Puerto-Rico	36	168170	9	0	0	40
+0	Private	Bachelors	Married-civ-spouse	Sales	Husband	White	Male	United-States	39	205338	13	0	0	60
+0	Self-emp-not-inc	7th-8th	Married-civ-spouse	Craft-repair	Husband	White	Male	United-States	44	98806	4	0	0	38
+0	State-gov	HS-grad	Married-civ-spouse	Other-service	Husband	White	Male	United-States	45	226452	9	0	0	40
+0	State-gov	11th	Never-married	Protective-serv	Own-child	White	Male	United-States	32	479179	7	0	0	40
+#@ TextLoader{
+#@   header+
+#@   sep=tab
+#@   col=Label:R4:0
+#@   col=Workclass:TX:1
+#@   col=Categories:TX:2-8
+#@   col=NumericFeatures:R4:9-14
+#@ }
+Label	Workclass	education	marital-status	occupation	relationship	ethnicity	sex	native-country-region	age	fnlwgt	education-num	capital-gain	capital-loss	hours-per-week
```

Comment:
There is another copy of the header within the test dataset. This could indicate an issue in its generation. We should look for duplicate rows within the file, and also duplicated between the train/test.